src/test/java/com/google/sps/RankTest.java

```diff
@@ -350,93 +529,118 @@ public void mixedPreferencesDoubleTie() {
     List<Schedule> expected = Arrays.asList(schedule3, schedule4);
     Assert.assertEquals(expected, actual);
   }
-
-  // TODO: Add all the tests for the big schedules, once merged into HandleRecitations branch
-
+    
   @Test
-  public void bigTestEmptyPreference() {
-    // RESTORE AFTER PRINTING 
-    // Preferences preferenceList = new Preferences(Arrays.asList());
-
-    // List<Schedule> schedules = Arrays.asList(bigSchedule1, bigSchedule2, bigSchedule3, bigSchedule4);
-    // preferenceList.sortSchedules(schedules);
-
-    // List<Schedule> actual = schedules;
-    // List<Schedule> expected = Arrays.asList(bigSchedule1, bigSchedule2, bigSchedule3, bigSchedule4);
-    // Assert.assertEquals(expected, actual);
-
-    // For printing purposes, will remove when merged into HandleRecitations branch
-    List<Course> courses1 = createBigScheduleCourses1();
-    Scheduler scheduler = new Scheduler();
-    List<Schedule> schedules1 = scheduler.generateSchedules(courses1, new Invariants(36, 60));
-    System.out.println(schedules1.size());
+  public void emptyPreferenceBigTest() {
+    Preferences preferenceList = new Preferences(Arrays.asList());
+    List<Schedule> schedules = Arrays.asList(bigSchedule1, bigSchedule2, bigSchedule3, bigSchedule4, bigSchedule5, bigSchedule6,
+                                             bigSchedule7, bigSchedule8, bigSchedule9, bigSchedule10, bigSchedule11, bigSchedule12,
+                                             bigSchedule13, bigSchedule14, bigSchedule15, bigSchedule16, bigSchedule17, bigSchedule18,
+                                             bigSchedule19, bigSchedule20, bigSchedule21, bigSchedule22, bigSchedule23, bigSchedule24);
+    preferenceList.sortSchedules(schedules);
 
-    for (Schedule schedule : schedules1) {
-      printSchedule(schedule);
-    }
+    List<Schedule> actual = schedules;
```

Comment:
I've seen this in a few tests, and I'm guessing this line was added for readability since `schedules` gets modified by `sortSchedules`. However in general I'd recommend not using an extra variable if not necessary. You can just put `Assert.assertEquals(expected, schedules)`.

Reply:
Ok sounds good. I can change it for every test